ProfilerGUI/Source/Configurator/MainViewModel.cs

```diff
@@ -0,0 +1,190 @@
+﻿using Newtonsoft.Json;
+using NLog;
+using ProfilerGUI.Source.Runner;
+using ProfilerGUI.Source.Shared;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using Common;
+
+namespace ProfilerGUI.Source.Configurator
+{
+    /// <summary>
+    /// The main view model to interact with the <see cref="MainWindow"/>.
+    /// </summary>
+    public class MainViewModel : INotifyPropertyChanged
+    {
+        private static readonly Logger logger = LogManager.GetCurrentClassLogger();
+
+        private static readonly string UploadConfigFilePath = Path.Combine(Directory.GetParent(AppDomain.CurrentDomain.BaseDirectory).FullName, "UploadDaemon", UploadConfig.ConfigFileName);
```

Comment:
we're writing to this file, hence I would not using the install dir but something in APP_DATA. If we want support distributing config files w/ profiles we should allow opening them.

Reply:
created #29 for this. I'm not 100% sure this is what we want. Let's discuss this after merging this